srv/components/provisioner/files/cluster_sls.template

```diff
@@ -30,15 +30,17 @@ cluster:
   dns_servers:                        # Do not update
   server_nodes:
     {% for num in range(num_of_nodes) %}
-    {{ grains['machine_id'] }}: srvnode-{{ (num + 1) }}
+    {% set node = 'srvnode-'  + (num + 1)|string %}
+    - {{ salt['mine.get'](node , 'node_machine_id')[node] }} : {{ node }}
     {% endfor %}
   {% for num in range(num_of_nodes) %}
+  {% set node = 'srvnode-'  + (num + 1)|string %}
```

Comment:
An extra space here:
`{% set node = 'srvnode-' + (num + 1)|string %}`

Reply:
comment is addressed